src/aac/aac.cc

```diff
@@ -319,7 +319,13 @@ static void aac_seek (VFSFile & file, NeAACDecHandle dec, int time, int len,
     unsigned char chan;
     unsigned long rate;
 
-    if ((used = NeAACDecInit (dec, (unsigned char *) buf, * buflen, & rate, & chan)))
+    if ((used = NeAACDecInit (dec, (unsigned char *) buf, * buflen, & rate, & chan)) < 0)
+    {
+        AUDERR ("Failed to initialize AAC decoder.\n");
+        return;
```

Comment:
If `NeAACDecInit()` returns an error, then is it still safe to proceed with calling `NeAACDecDecode()`? If not, perhaps we should make `aac_seek()` return an error in that case so that `AACDecoder::play()` can break out of the decode loop.

Reply:
Good catch. Can we use `* buflen = 0;` like a few lines above when checking the result of `aac_probe()`?

Reply:
Looks like that should work, yes.